docs/src/components/home/DesignKits.tsx

```diff
@@ -40,41 +44,64 @@ const Anchor = styled('a')({
   },
 });
 
-const DesignTool = React.forwardRef<HTMLAnchorElement, { brand: 'figma' | 'sketch' | 'xd' }>(
-  ({ brand, ...props }, ref) => {
-    return (
-      <Anchor
-        ref={ref}
-        aria-label="Goto MUI store"
-        href={{ figma: ROUTES.storeFigma, sketch: ROUTES.storeSketch, xd: ROUTES.storeXD }[brand]}
-        target="_blank"
-        {...props}
-      >
-        <Avatar src={`/static/branding/design-kits/designkits-${brand}.png`} alt="" />
-        <Box
-          sx={{
-            transition: '0.3s',
-            borderRadius: '50%',
-            position: 'absolute',
-            width: '100%',
-            height: '100%',
-            opacity: 0,
-            top: 0,
-            left: 0,
-            bgcolor: (theme) => alpha(theme.palette.primary[500], 0.5),
-            color: '#fff',
-            display: 'flex',
-            flexDirection: 'column',
-            alignItems: 'center',
-            justifyContent: 'center',
-          }}
-        >
-          <LaunchRounded />
-        </Box>
-      </Anchor>
-    );
-  },
-);
+const DesignToolLink = React.forwardRef<
+  HTMLAnchorElement,
+  React.PropsWithChildren<{ brand: 'figma' | 'sketch' | 'xd' }>
+>(({ brand, ...props }, ref) => (
+  <Anchor
+    ref={ref}
+    aria-label="Goto MUI store"
+    href={{ figma: ROUTES.storeFigma, sketch: ROUTES.storeSketch, xd: ROUTES.storeXD }[brand]}
+    target="_blank"
+    {...props}
+  >
+    {props.children}
+    <Box
+      sx={{
+        transition: '0.3s',
+        borderRadius: '50%',
+        position: 'absolute',
+        width: '100%',
+        height: '100%',
+        opacity: 0,
+        top: 0,
+        left: 0,
+        bgcolor: (theme) =>
+          theme.palette.mode === 'dark'
+            ? alpha(theme.palette.primaryDark[800], 0.5)
+            : alpha(theme.palette.primary[500], 0.5),
+        color: '#fff',
```

Comment:
Same as the prev question, is this fine for both dark and light mode?

Reply:
in this case, yes.

Reply:
To confirm with @danilo-leal  because this is not in the design but I think it is useful and Olivier has the same experience that it is clickable.

<img width="466" alt="Screen Shot 2564-08-23 at 19 50 50" src="https://user-images.githubusercontent.com/18292247/130450242-09eced51-a1c8-4711-9210-991df231b80e.png">
<img width="461" alt="Screen Shot 2564-08-23 at 19 50 56" src="https://user-images.githubusercontent.com/18292247/130450256-eb6fc8a7-582a-434e-81cd-1e78ee95f174.png">

Reply:
It's fine for me! (Not having the circle divs become dark and having them as links).